### DISCUSSION TITLE
Exclude files in the api folder the same way as the pages folder

### DISCUSSION THREAD

**Discussion by raae** [gatsbyjs/gatsby#33833] [Ideas / Feature Requests] [answered]
When adding test files to the api folder they become endpoints.

<img width="480" alt="CleanShot 2021-11-03 at 08 57 59@2x" src="https://user-images.githubusercontent.com/478531/140028600-2ffbd4ad-2da9-49de-955d-41e76a63433e.png">

I would expect it to be excluded in similar fashion as files in the pages folder.

**Comment by LekoArts** [comment, marked as answer]
Hi!

Yeah, you're right, it should not convert those into functions. I'll put up a PR :)

**Reply by LekoArts** [reply]
PR is up: https://github.com/gatsbyjs/gatsby/pull/33834

**Reply by LekoArts** [reply]
Will be released in Gatsby 4.2
